### ISSUE TITLE
Clean old note instead of archive

### ISSUE THREAD

**Issue by DHERRADOR** [Newbrict/obsidian-obligator#29]
(feature request)

Dear Dimitar,

 Thank you very much for your plugin.

I wanted to suggest you if there would be an option to configure in Obligator, the option to delete the unclosed tasks of the old note.
That is, something similar to what Rollover does.

My workflow is to leave written in "Calendar", within each day, the tasks that I have been doing. That is, each day I have "Closed tasks" in the past.

And I am only interested in having open tasks in the present and future.

For me it's not useful that when I use Obsidian's search engine, if I search for a task, it comes up 20 different days. 19 in the Archive folder, and 1 in the Calendar folder.
And I don't see feasible to exclude the Archive folder from the search, because the way Obligator works today, I would have to check there when I finished a certain task. Since Obligator would archive them there.

In short, as Obligator works today, I have to disable the option to "archive" the old note. And therefore, as it repeats the open tasks in the new day's note, I have to manually go to the previous day to "clean up" the open tasks that I didn't have time to do.

Why I don't use rollover plugin, as it works as I need, because it doesn't have the super useful "Template macros" option.

Please consider this option.
It would also help me to use Rollovel plugin to send my old tasks to the present, to the current day note, if I could "disable in Ogligator" the functionality to move the old tasks. That is, to use only the "Template macro" in Obligator. But honestly, I would prefer the first option as I find your plugin more reliable.

I hope you can consider my suggestion.

Best regards.

**Comment by Newbrict** [comment]
That is an interesting suggestion, I will keep it in mind. I'm not going to start work on it now, but next time I'm working on Obligator I will try to get around to it. Thanks.

**Comment by brdweb** [comment]
I'd second this suggestion. I like to use the tasks calendar wrapper so that I have my tasks open when I'm looking at other notes and it will find all copies of a task. Right now I'd have to go back and close every single copy in order to clean that sidebar up. 

One other way would be to MOVE the task to the new daily note instead of copying it, thus only keeping a single copy.
